### ISSUE TITLE
Cannot ignore a file

### ISSUE THREAD

**Issue by rwb196884** [prettier/prettier#14204]
I'm trying to ignore a file genrated from Swagger, but this appears to be impossible.

**.prettierignore**

```
ignored_file.ts
```

**What happens**
```
$ prettier --config .prettierrc --ignore-path .prettierignore --check "src/**/*.{js,ts,tsx}"
1>  Checking formatting...
1>  src\ignored_file.ts
1>  Code style issues found in the above file(s). Forgot to run Prettier?
1>  error Command failed with exit code 1.
```

Is it possible to ignore a file?

**Comment by fisker** [comment]
What's in `.prettierignore`?

**Comment by rwb196884** [comment]
It was set up with version 1.19.1 and I think this has been fixed by 2.8.3.
